discovery_engine/test/ffi/types/document/document_test.dart

```diff
@@ -49,4 +51,32 @@ void main() {
     ffi.drop_document(place);
     expect(res, equals(document));
   });
+
+  test('conversion to Document works', () {
+    final ffiDocument = DocumentFfi(
```

Comment:
iirc i've seen those setups in other tests as well, should we place common setups in some test utils module?